datahub/main.py

```diff
@@ -85,3 +96,34 @@ def update_dsr_data(data: DSRModel) -> dict[str, str]:
     dt.dsr_data.append(data_dict)
 
     return {"message": "Data submitted successfully."}
+
+
+@app.get("/dsr")
+def get_dsr_data(  # type: ignore[misc]
+    start: int = 0, end: int | None = None
+) -> dict[Hashable, Any]:
+    """GET method function for getting DSR data as JSON.
+
+    Returns:
+        A Dict containing the DSR list.
+    """
+    filtered_data = dt.dsr_data[start:]
+
+    if end:
+        filtered_data = filtered_data[: end - start + 1]
+
+    return {"data": filtered_data}
+
+
+@app.get("/wesim")
+def get_wesim_data() -> dict[Hashable, Any]:  # type: ignore[misc]
```

Comment:
This function doesn't look quite right, and since we're waiting on #57 I say remove it from this PR for now and tackle it after.